### DISCUSSION TITLE
Follow upgrade of Django Ninja

### DISCUSSION THREAD

**Discussion by lewoudar** [eadwinCode/django-ninja-extra#72] [General]
Hi,
I just want to know if you are aware of the recent changes in [Django Ninja](https://github.com/vitalik/django-ninja/releases/tag/v1.0a1) towards Pydantic V2 support and if we will have it soon in extras? ^^

I'm also wondering if there will be an evolution for [ninja-schema](https://pypi.org/project/ninja-schema/).

**Comment by eadwinCode** [comment]
@lewoudar sorry for late response. I am aware of the pydantic v2 upgrade. Its a big work that requires time. But I will do the migration on django-ninja-extra, django-ninja-jwt and ninja-schema in due time. Django ninja pydantic v2 support is still on alpha. I believe in due time a stable version will emerge.

**Reply by lewoudar** [reply]
No worries, I know there is a lot of work ;)

I just want to be sure you were aware of django-ninja development :)

**Reply by eadwinCode** [reply]
Alright
